crates/newtype-uuid/src/lib.rs

```diff
@@ -277,8 +284,7 @@ impl TypedUuidTag {
     ///
     /// # Panics
     ///
-    /// Panics if the above conditions aren't met. (This is a const fn, so it can't return an
-    /// error.)
+    /// Panics if the above conditions aren't met. Use [`Self::try_new`] to handle errors instead.
     pub const fn new(tag: &'static str) -> Self {
```

Comment:
this should probably also have a `#[must_use]` on it
```suggestion
    #[must_use]
    pub const fn new(tag: &'static str) -> Self {
```

Reply:
Ahh yes, thanks!